iina/LogWindowController.swift

```diff
@@ -83,9 +97,9 @@ class LogWindowController: NSWindowController, NSTableViewDelegate, NSTableViewD
   }
 
   @IBAction func save(_ sender: Any) {
-    Utility.quickSavePanel(title: "Log", filename: "log.txt", sheetWindow: window) { URL in
-      let logs = (self.logArrayController.arrangedObjects as! [Log]).map { $0.logString }.joined()
-      try? logs.write(to: URL, atomically: true, encoding: .utf8)
+    Utility.quickSavePanel(title: "Log", filename: "log.txt", sheetWindow: window) { url in
```

Comment:
I am surprised the default filename, `log.txt`, is not the "real" log filename, `iina.log`.

Reply:
I confirmed this has been addressed.